### ISSUE TITLE
ratios for the very long chain fatty acids

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7744]
**Preferred term label:**
see https://pubmed.ncbi.nlm.nih.gov/27557811/

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
617370 
PMID: 27557811  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
let's hold off on this for now; we now have many more acylcarnitine terms and these ratios are not very commonly used.
